samples/resilience-examples/src/test-util.ts

```diff
@@ -0,0 +1,33 @@
+import { setTestDestination } from '@sap-cloud-sdk/test-util';
+import nock from 'nock';
+import {
+  BusinessPartner,
+  businessPartnerService
+} from '@sap/cloud-sdk-vdm-business-partner-service';
+
+export const destinationName = 'MyDestination';
+export const destinationUrl = 'http://some.url.com';
+
+export function mockDestination(): void {
+  setTestDestination({ name: destinationName, url: destinationUrl });
+}
+
+export const sampleResponse = {
+  d: {
+    results: [{ BusinessPartner: 1234 }]
+  }
+};
+
+export function mockGetAllRequest(times: number): void {
+  nock(destinationUrl).get(/.*/).times(times).reply(200, sampleResponse);
+}
+
+export async function getAllBusinessPartner(
+  top: number
+): Promise<BusinessPartner[]> {
```

Comment:
It's a bit confusing to me that this is part of `test-util.ts`. Also I would like to rename the parameter to something that is clearer without knowing the implementation (e.g. `filterTop` or maybe `filterTopBuPa`).

Reply:
hm, about the location of the method I agree. Perhaps just copy past it twice?